container/providers.bzl

```diff
@@ -20,7 +20,11 @@ BundleInfo = provider(fields = ["container_images", "stamp"])
 FlattenInfo = provider()
 
 # A provider containing information exposed by container_image rules
-ImageInfo = provider(fields = ["container_parts"])
+ImageInfo = provider(fields = [
+    "container_parts",
```

Comment:
Why not make these part of container_parts?

Reply:
I feel container_parts apply more to the static image tarball itself. But the run flags are more about the runtime?